### PR TITLE
Document AUTHORIZED_FETCH and WHITELIST_MODE

### PR DESCRIPTION
**Pull request by ThibG**

### CODE REVIEW

File: content/en/admin/config.md [hunk 1]
```diff
@@ -22,6 +22,24 @@ For convenience, it can read them from a flat file called `.env.production` in t
 * `LOCAL_DOMAIN`
 * `WEB_DOMAIN`
 * `ALTERNATE_DOMAINS`
+* `AUTHORIZED_FETCH`
```
Comment (Gargron) [inline content/en/admin/config.md]: If these become subheaders instead they'll be linkable

Comment (ThibG) [reply content/en/admin/config.md]: should we change… all of them, then? :/

Comment (Gargron) [reply content/en/admin/config.md]: Just the ones that have content, in my opinion (these 2 new ones)

Comment (ThibG) [reply content/en/admin/config.md]: :woman_shrugging:

**Pull request merged**